services/alertmanager/alertmanager.go

```diff
@@ -35,12 +35,17 @@ import (
 )
 
 const (
-	resendInterval      = 30 * time.Second
-	alertmanagerDataDir = "/srv/alertmanager/data"
-	prometheusDir       = "/srv/prometheus"
-	path                = "/srv/alertmanager/alertmanager.base.yml"
+	// Environment variable to overwrite resendInterval during testing
+	envInterval = "PERCONA_TEST_ALERTMANAGER_INTERVAL"
```

Comment:
Let's call it `envResendInterval` and `PERCONA_TEST_ALERTMANAGER_RESEND_INTERVAL` to match `resendInterval` below.